third_party/openthread/src/ncp/spinel.h

```diff
@@ -227,6 +227,16 @@ enum
     SPINEL_MAC_PROMISCUOUS_MODE_FULL     = 2, ///< All decoded MAC packets are passed up the stack.
 };
 
+enum
+{
+    SPINEL_NCP_LOG_LEVEL_NONE  = 0,
+    SPINEL_NCP_LOG_LEVEL_CRIT  = 1,
+    SPINEL_NCP_LOG_LEVEL_WARN  = 2,
+    SPINEL_NCP_LOG_LEVEL_INFO  = 3,
+    SPINEL_NCP_LOG_LEVEL_DEBG  = 4,
+    SPINEL_NCP_LOG_LEVEL_MAX   = SPINEL_NCP_LOG_LEVEL_DEBG,
+};
```

Comment:
The definition here maps to the current log levels defined in OT.

If we go to syslog enumeration then, we need to have a way to convert them to OT log level (which has fewer levels).